<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title-template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="connectBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="connectionStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Нет подключения</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>160</width>
       <height>199</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QRadioButton" name="regButton">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="authButton">
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Авторизация</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="loginLine"/>
          </item>
          <item>
           <widget class="QLineEdit" name="pswdLine"/>
          </item>
          <item>
           <widget class="QPushButton" name="execButton">
            <property name="text">
             <string>Выполнить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Подключиться</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="answerBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>242</width>
       <height>134</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ваш ответ:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="taskLine"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Номер варианта:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="sendTaskButton">
          <property name="text">
           <string>Отправить на проверку</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelAnswerStatus">
          <property name="text">
           <string>Статус ответа:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="answerStatusLabel">
          <property name="text">
           <string>Неизвестно</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>Авторизация</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
